blueocean-executor-info-plugin/Jenkinsfile

```diff
@@ -0,0 +1,3 @@
+/* defined here: https://github.com/jenkins-infra/pipeline-library/ */
```

Comment:
I think we can delete this file since the root compiles everything.

Reply:
Done.